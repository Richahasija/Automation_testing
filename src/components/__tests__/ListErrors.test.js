// Link.react.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import ListErrors from '../ListErrors';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <ListErrors errors={['errror']}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});