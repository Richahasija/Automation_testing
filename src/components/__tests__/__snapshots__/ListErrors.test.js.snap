// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<ul
  className="error-messages"
>
  <li>
    0
     
    errror
  </li>
</ul>
`;
